tools/shared/src/main/scala/org/scalajs/core/tools/linker/frontend/BaseLinker.scala

```diff
@@ -337,7 +337,8 @@ final class BaseLinker(semantics: Semantics, esLevel: ESLevel, considerPositions
       analysis: Analysis): MethodDef = {
     val encodedName = methodInfo.encodedName
 
-    val targetMDef = findInheritedMethodDef(classInfo, targetName, getTree)
+    val targetMDef = findInheritedMethodDef(analysis, classInfo, targetName,
+        getTree)
```

Comment:
I do not understand why we need this. Isn't the `targetName` resolved to something that actually implements the method (as in: not via synthetic method). So we should be able to just call it there.